buffalo_poc/actions/app.go

```diff
@@ -0,0 +1,71 @@
+package actions
+
+import (
+	"github.com/gobuffalo/buffalo"
+	"github.com/gobuffalo/buffalo/middleware"
+	"github.com/gobuffalo/buffalo/middleware/ssl"
+	"github.com/gobuffalo/envy"
+	"github.com/unrolled/secure"
+
+	"github.com/dahendel/kore-poc/buffalo_poc/models"
+	"github.com/gobuffalo/buffalo/middleware/csrf"
+	"github.com/gobuffalo/buffalo/middleware/i18n"
+	"github.com/gobuffalo/packr"
+)
+
+// ENV is used to help switch settings based on where the
+// application is being run. Default is "development".
+var ENV = envy.Get("GO_ENV", "development")
+var app *buffalo.App
```

Comment:
Out of curiosity, why not make this the first line of the App() function?

Reply:
On second glance, this looks like something buffalo just does...I'm still curious as to why but this doesn't need to change. 🤷‍♂️